images/v2v-job/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM fedora:26
```

Comment:
Does it make sense to have a Makefile target (at the top level) for building this image?  The reason I ask is when we add the templates/ dir at the top level also we will need to exec this build there using 
```
  docker build -f images/v2v-job/Dockerfile .
```